packages/react-components/react-components/.storybook/main.utils.js

```diff
@@ -6,14 +6,17 @@
 const fs = require('fs');
 const path = require('path');
 
+// Dependencies to exclude stories loading
+const excludedDependencies = ['@fluentui/react-overflow'];
```

Comment:
probably out of topic, but Im confused a bit 😅,  AFAIK we use public-docsite-v9 to build and ship storybook to public, so this shouldn't be needed ?

Reply:
If I correctly understood the question: we don't want to show stories for this package (yet?), but we would like to re-export it, https://github.com/microsoft/fluentui/pull/23226#pullrequestreview-984614791.

Reply:
I know, that's the purpose of this PR. I was rather asking why do we need to do this in 2 places
- react-components
- public-docsite-v9

Reply:
ah you're correct, I should have only done it in the public docsite

Reply:
However I'm still not entirely sure of the boundaries there. If we don't consider anything in react-components storybook to be public docsite anymore we can remove it

Reply:
> If we don't consider anything in react-components storybook to be public docsite anymore we can remove it

agreed